src/internal/visual-mode/__tests__/use-runtime-visual-refresh.test.tsx

```diff
@@ -18,8 +18,10 @@ describe('useVisualRefresh', () => {
     return <div data-testid="current-mode">{isRefresh.toString()}</div>;
   }
 
-  beforeEach(() => clearVisualRefreshState());
-  afterEach(() => document.body.classList.remove('awsui-visual-refresh'));
```

Comment:
Why was this not captured before – we cleaned it up in the unit tests manually

When I tried to use this utility in cloudscape-design/components, I found that it does not reset to classic properly